marimbabot_audio/README.md

```diff
@@ -0,0 +1,26 @@
+# Sound2Note
+
+## 1. Introduction 
+This is a RosNode to translate the sound signal to western notation, based on the base version from Michael. 
+## 2. Installation
+### 2.1 Declaration
+This installation instruction has been currently verified only at the lab compute. 
+
+### 2.2 Installation of python virtual environment
+1. Come to your project position, the build the python virtualenv: `python -m venv env_sound2note --system-site-packages && source env_sound2note/bin/activate`
+2. Build you ros workspace: `mkdir -p ws_soud2note/src && cd ws_soud2note/src`
+3. Download the code: `git clone https://github.com/yunlong-wang-cn/music_perception.git`
+4. Install the python requirements: `pip install librosa`, and the `pip install crepe`
```

Comment:
Can you add them to the `requirements.txt` of this repo instead?

Reply:
done